### DISCUSSION TITLE
Should we move to Laravel for the backend?

### DISCUSSION THREAD

**Discussion by StJudeWasHere** [wwwdevcc/projects#6] [Ideas]
A few of us have been talking about moving the backend to Laravel and the front end to React.
I think it's a good call and would help as have something more structured. Also many developers feel much more confident using these frameworks than just plain php.

We could also opt for something a bit more minimal, like slimphp, as it would still be simple enough but would help organizing things a bit.

If we keep adding features to what we have now, it will certainly become a difficult to work with. So I'd love to hear your opinions on this.

**Comment by FredMT** [comment]
I am personally in favor of this motion! I and a lot more people would be able to contribute a lot more immediately we have a React frontend. Laravel backend sounds like a great idea too.
